code/modules/clothing/under/miscellaneous.dm

```diff
@@ -1197,3 +1197,39 @@ Uniforms and such
 	name = "hazard cohesion suit"
 	desc = "An orange cohesion suit with yellow hazard stripes intended to assist Prometheans in maintaining their form and prevent direct skin exposure."
 	icon_state = "cohesionsuit_hazard"
+
+/obj/item/clothing/under/utility/supply
+	name = "supply utility uniform"
+	desc = "A comfortable turtleneck and black utility trousers. This one has accents in the supply department's color."

```

Comment:
I believe cerebul had words about this description being odd paired with the sprite, so please correct that as well.